### PR TITLE
Feat:Html-media Issue #118

### PR DESCRIPTION
**Pull request by zainbinshakoor**
@Diptenusarkar 
- #118

### CODE REVIEW

File: html/html_media/index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                Media elements in HTML enrich web content by allowing the integration of audio, video, and external content. They enhance user engagement and interactivity.
+            </p>
+            <hr>
+            <h2>Audio (Using &lt;audio&gt;)</h2>
+            <p>
+                The &lt;audio&gt; element enables you to embed audio content like music or podcasts. You can specify the audio source using the "src" attribute and customize playback controls.
+                <br>
+                Example:
+                <pre class="language-html">
+        &lt;audio src="music.mp3" controls&gt;&lt;/audio&gt;
```
Comment (Diptenusarkar) [inline html/html_media/index.html:107]:
```suggestion
<code>
        &lt;audio src="music.mp3" controls&gt;&lt;/audio&gt;
        </code>
```

File: html/html_media/index.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                </pre>
+            </p>
+            <h2>Video (Using &lt;video&gt;)</h2>
+            <p>
+                The &lt;video&gt; element seamlessly embeds videos into webpages. It supports various formats (e.g., MP4, WebM) for broader compatibility. Customize playback controls, dimensions, and multiple source formats.
+                <br>
+                Example:
+                <pre class="language-html">
+        &lt;video width="640" height="360" controls&gt;
```
Comment (Diptenusarkar) [inline html/html_media/index.html:116]:
``` suggestion
<code>
        &lt;video width="640" height="360" controls&gt;
```

File: html/html_media/index.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            &lt;source src="video.mp4" type="video/mp4"&gt;
+            &lt;source src="video.webm" type="video/webm"&gt;
+            Your browser does not support the video tag.
+        &lt;/video&gt;
```
Comment (Diptenusarkar) [inline html/html_media/index.html:120]:
```suggestion
        &lt;/video&gt;
        </code>
```

File: html/html_media/index.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+                </pre>
+            </p>
+            <h2>Embedding YouTube Videos</h2>
+            <p>
+                Use an &lt;iframe&gt; element with the YouTube video URL to easily embed and play YouTube videos on your website.
+                <br>
+                Example:
+                <pre class="language-html">
+        &lt;iframe width="560" height="315" src="https://www.youtube.com/embed/VIDEO_ID" frameborder="0" allowfullscreen&gt;&lt;/iframe&gt;
```
Comment (Diptenusarkar) [inline html/html_media/index.html:129]:
```suggestion
<code>
        &lt;iframe width="560" height="315" src="https://www.youtube.com/embed/VIDEO_ID" frameborder="0" allowfullscreen&gt;&lt;/iframe&gt;
```

File: html/html_media/index.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+                </pre>
```
Comment (Diptenusarkar) [inline html/html_media/index.html:130]:
```suggestion
</code>
                </pre>
```

**Review by Diptenusarkar — CHANGES_REQUESTED** [review]
Please comply with this.

**Comment by zainbinshakoor** [comment]
@Diptenusarkar could you explain it a little bit more?

**Comment by zainbinshakoor** [comment]
it will not count from mine side?
@Diptenusarkar

**Comment by zainbinshakoor** [comment]
@Diptenusarkar
 Could you please review this one issue[https://github.com/Diptenusarkar/Website-Template/issues/367](url)

**Pull request merged**